hack/test-cmd.sh

```diff
@@ -428,6 +428,39 @@ runTests() {
   # Post-condition: valid-pod POD has image kubernetes/pause
   kube::test::get_object_assert pods "{{range.items}}{{$image_field}}:{{end}}" 'kubernetes/pause:'
 
+  ## If resourceVersion is specified in the patch, it will be treated as a precondition, i.e., if the resourceVersion is different from that is stored in the server, the Patch should be rejected
+  ERROR_FILE="${KUBE_TEMP}/conflict-error"
+  ## If the resourceVersion is the same as the one stored in the server, the patch will be applied.
+  # Command
+  # Needs to retry because other party may change the resource.
+  for count in $(seq 0 3); do
```

Comment:
@JanetKuo, this tests patch would be accepted if the resourceVersion is correct. PTAL. Thank you.